cabal-install/Distribution/Client/CmdShowBuildInfo.hs

```diff
@@ -0,0 +1,327 @@
+-- | cabal-install CLI command: new-show-build-info
+--
+module Distribution.Client.CmdShowBuildInfo where
+-- (
+--     -- * The @show-build-info@ CLI and action
+--     showBuildInfoCommand,
+--     showBuildInfoAction
+--   )
+
+import Distribution.Client.Compat.Prelude
+         ( when, find, fromMaybe )
+import Distribution.Client.ProjectOrchestration
+import Distribution.Client.CmdErrorMessages
+import Distribution.Client.CmdInstall.ClientInstallFlags
+
+import Distribution.Client.Setup
+         ( GlobalFlags, ConfigFlags(..), ConfigExFlags, InstallFlags
+         )
+import qualified Distribution.Client.Setup as Client
+import Distribution.Simple.Setup
+         ( HaddockFlags, TestFlags
+         , fromFlagOrDefault
+         )
+import Distribution.Simple.Command
+         ( CommandUI(..), option, reqArg', usageAlternatives
+         )
+import Distribution.Verbosity
+         ( Verbosity, silent )
+import Distribution.Simple.Utils
+         ( wrapText, die', withTempDirectory )
+import Distribution.Types.UnitId
+         ( UnitId, mkUnitId )
+import Distribution.Types.Version
+         ( mkVersion )
+import Distribution.Types.PackageDescription
+         ( buildType )
+import Distribution.Deprecated.Text
+         ( display )
+
+import qualified Data.Map as Map
+import qualified Distribution.Simple.Setup as Cabal
+import Distribution.Client.SetupWrapper
+import Distribution.Simple.Program ( defaultProgramDb )
+import qualified Distribution.Client.InstallPlan as InstallPlan
+import Distribution.Client.ProjectPlanning.Types
+import Distribution.Client.ProjectPlanning (
+  setupHsConfigureFlags, setupHsConfigureArgs,
+  setupHsBuildFlags, setupHsBuildArgs,
+  setupHsScriptOptions
+  )
+import Distribution.Client.DistDirLayout (distBuildDirectory)
+import Distribution.Client.Types ( PackageLocation(..), GenericReadyPackage(..) )
+import Distribution.Client.JobControl (newLock, Lock)
+import Distribution.Simple.Configure (tryGetPersistBuildConfig)
+import qualified Distribution.Client.CmdInstall as CmdInstall
+
+import System.Directory (getTemporaryDirectory)
+import System.FilePath ((</>))
+
+showBuildInfoCommand :: CommandUI ShowBuildInfoFlags
+showBuildInfoCommand = CmdInstall.installCommand {
+  commandName         = "new-show-build-info",
```

Comment:
This should probably just be "show-build-info", right?

Reply:
I think so, the default was already switched to v2 in 3.0.